src/views/templates/details/tabs/details/components/BootOrderItem.tsx

```diff
@@ -17,11 +22,33 @@ type BootOrderProps = {
 const BootOrderItem: React.FC<BootOrderProps> = ({ template }) => {
   const { t } = useKubevirtTranslation();
   const disks = getTemplateDisks(template);
+  const history = useHistory();
   const interfaces = getTemplateInterfaces(template);
+  const onDisksTab = () =>
+    history.push(
+      `/k8s/ns/${template.metadata.namespace}/templates/${template.metadata.name}/disks`,
+    );
 
   return (
     <DescriptionListGroup>
-      <DescriptionListTerm>{t('Boot order')}</DescriptionListTerm>
+      <DescriptionListTermHelpText>
+        <Popover
+          hasAutoWidth
+          maxWidth="15rem"
+          position="right"
+          bodyContent={
+            <Trans ns="plugin__kubevirt-plugin">
+              You can edit the boot order in the{' '}
+              <Button type="button" isInline onClick={onDisksTab} variant="link">
```

Comment:
Hi
I think you can simply use Link from react-router-dom instead of a button and I click method with push :)